src/platforms/common/configuration/filtering.mdx

```diff
@@ -98,3 +82,49 @@ In this example, the fingerprint is forced to a common value if an exception of
 : For breadcrumbs created from HTTP requests done via the legacy `XMLHttpRequest` API. This holds the original xhr object.
 
 <PlatformContent includePath="configuration/decluttering" />
+
+## Sampling Error Events
+
+To send a representative sample of your errors to Sentry, set the `sampleRate` option in your SDK configuration to a number between `0` (0% of errors sent) and `1` (100% of errors sent). This is a static rate, which will apply equally to all errors. For example, to sample 25% of your errors:
```

Comment:
```suggestion
To send a representative sample of your errors to Sentry, set the `sampleRate` option in your SDK configuration to a number between `0` (0% of errors sent) and `1` (100% of errors sent). This is a static rate, which will apply equally to all errors. Since the error sample rate is not dynamic, changing it requires re-deployment. In addition, setting an SDK sample rate limits visibility into the source of events. Setting a rate limit for your project (which only drops events when volume is high) may better suit your needs.

For example, to sample 25% of your errors:
```